cmake/ProjectWabt.cmake

```diff
@@ -22,10 +22,10 @@ ExternalProject_Add(wabt
     CMAKE_ARGS
     -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
     -DCMAKE_BUILD_TYPE=Release
-    -DWITH_EXCEPTIONS=ON
+    -DWITH_EXCEPTIONS=OFF
```

Comment:
Why does this needs to be turned off?

Reply:
It matches wabt default build options.
This breaks some EEI. Long term it would be worth to re-implement EEI without exceptions.